gcloud-java-examples/README.md

```diff
@@ -37,7 +37,8 @@ To run examples from your command line:
 
 2. Set your current project using `gcloud config set project PROJECT_ID`. This step is not necessary for `ResourceManagerExample`.
 
-3. Compile using Maven: `cd gcloud-java-examples` in command line from your base project directory and then `mvn package appassembler:assemble -DskipTests -Dmaven.javadoc.skip=true  -Dmaven.source.skip=true`.
+3. Compile using Maven: `mvn install -DskipTests -Dmaven.javadoc.skip=true -Dmaven.source.skip=true` in command line from your base project directory
```

Comment:
We recently added a [release profile](https://github.com/GoogleCloudPlatform/gcloud-java/pull/889), javadoc and source building have been disabled unless the release profile is used (`-P release`). So i believe you can remove the `-Dmaven.javadoc.skip=true -Dmaven.source.skip=true` options (also in the following command).